logger.gd

```diff
@@ -703,16 +727,16 @@ func load_config(configfile = default_configfile_path):
 	max_memory_size = config.get_value(PLUGIN_NAME, "max_memory_size", max_memory_size)
 
 	# Load logfiles config and initialize them
-	logfiles = {}
+	external_sinks = {}
 	for logfile_cfg in config.get_value(PLUGIN_NAME, "logfiles"):
```

Comment:
You changed the value to `external_sinks` when setting, but not when loading, so isn't this broken?

Reply:
I guess I did not use persisting the config and I did not notice.
Let me check!

Reply:
This should be fixed in 028a678